### ISSUE TITLE
Error in batch render test

### ISSUE THREAD

**Issue by dgriffiths3** [BachiLi/redner#146]
When running the batch test script the line:

```
h, w = args_old_format[11]
```
should be:
```
h, w = args_old_format[12]
```

Has there been a change to break this recently? The rest of the script runs fine, but I wonder if there could be other underlying issues with the other args as a result of being shifted 1 place that aren't so obvious.

**Comment by BachiLi** [comment]
Thanks for catching this! You're correct. In general this function is outdated -- there's a wrapper in `pyredner/render_utils.py` that should handle batch rendering. I should probably remove this file.

**Comment by dgriffiths3** [comment]
Okay thanks. So is the correct way to do batch rendering to just simply pass in a list of scenes to the deferred/generic rendering function?

**Comment by BachiLi** [comment]
Yes. In the future we may have more efficient ways to do this -- still figuring out.
